include/ast.hpp

```diff
@@ -93,6 +93,18 @@ struct NullStmtNode : public StmtNode {
   virtual void Accept(ModifyingVisitor&) override;
 };
 
+struct IfStmtNode : public StmtNode {
+  IfStmtNode(std::unique_ptr<ExprNode> expr,
+             std::unique_ptr<BlockStmtNode> block)
+      : expr{std::move(expr)}, block{std::move(block)} {}
+
+  virtual void Accept(NonModifyingVisitor&) const override;
+  virtual void Accept(ModifyingVisitor&) override;
+
+  std::unique_ptr<ExprNode> expr;
+  std::unique_ptr<BlockStmtNode> block;
```

Comment:
```suggestion
  std::unique_ptr<BlockStmtNode> body;
```
Suggest naming based on semantics rather than types.
Additionally, for future _if statement_ that incorporates both _then_ and _else_ parts, using `then` and `else` for naming would be a considerable choice.